### DISCUSSION TITLE
insert 时候有一个字段需要用sql里面的函数做插入  怎么自定义sql语句

### DISCUSSION THREAD

**Discussion by lhtuling** [dotnetcore/FreeSql#917] [Q&A]
如题
对接其他系统  数据库CreateTime字段用的时间戳   插入数据时候需要自动使用数据库当前时间
unix_timestamp(current_timestamp(3))*1000
可以实现但是不知道怎么使用

**Comment by 2881099** [comment]
实体特性，\[Column(RewriteSql = "unix_timestamp(current_timestamp(3))*1000")]
